static/src/javascripts/projects/common/modules/email/email.js

```diff
@@ -210,6 +211,19 @@ define([
                 });
 
             },
+            setTone: function ($el) {
+                var toneClass = 'tonal__main--tone-',
+                    toneClassRegexp = new RegExp('(' + toneClass + ')([a-z-]+)'),
+                    tone;
+
+                if ($el.hasClass('js-email-sub--article')) {
+                    // Get tone from class
+                    tone = $('.tonal__main')[0].className.match(toneClassRegexp)[2];
```

Comment:
Idea: I wonder if we should use `Option` in JavaScript. There are two values here that could be undefined: the element and the result of the regexp match. `Option` allows you to declare a value may be undefined and easily handle those cases.

I'm just thinking out loud, ignore me!